Makefile

```diff
@@ -225,7 +231,7 @@ run: generate fmt lint ## Run a controller from your host.
 
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen: ## Download controller-gen locally if necessary.
-	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.1)
+	$(call go-install-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0)
```

Comment:
we probably don't need to update controller-gen at least in this PR, WDYT ?

Reply:
yes I will revert that. Is there a followup somewhere ? I feel we should keep up to date on these.